contrib/pyln-proto/pyln/proto/zbase32.py

```diff
@@ -1,4 +1,4 @@
-import bitstring  # type: ignore
```

Comment:
Does `bitstring` now come with type annotations?

Reply:
i am using it here https://github.com/lnbits/bolt11 and neither mypy, pyright nor ruff are not complaining :).

Reply:
i had type ignores all over the place in bolt11 before i updated to ^4

Reply:
From the `bitsting` release notes: ` Type hints added throughout the code.` for `v4.0.1`